.eslintignore

```diff
@@ -7,3 +7,4 @@ ssl
 stacktrace*
 tmp
 sql
+docs/jsdoc
```

Comment:
EOF newline - do you have a plugin for editorconfig?

Reply:
installed it now :)